examples/HSDS/README.rst

```diff
@@ -0,0 +1,155 @@
+Highly Scalable Data Service (HSDS)
+===================================
+
+The Highly Scalable Data Service (HSDS) is a cloud-optimized solution for
+storing and accessing HDF5 files, e.g. the NREL wind and solar datasets. You
+can access NREL data via HSDS in a few ways. Read below to find out more.
+
+NREL Developer API
+------------------
+
+The easiest way to get started with HSDS is to get a developer API key via the
+`NREL Developer Network <https://developer.nrel.gov/signup/>`_. Once you have
+your API key, create an HSDS config file at ``~/.hscfg`` with the following
+entries (make sure you update the ``hs_api_key`` entry):
+
+.. code-block:: bash
+
+  # NREL dev api
+  hs_endpoint = https://developer.nrel.gov/api/hsds
+  hs_username =
```

Comment:
Do they need to fill in their username/password too, or should they leave it blank like this?

Reply:
No need for un/pw. You actually don't need these lines, they're an artifact of using the hsconfig tool. Will remove.